programs/ssl/ssl_client2.c

```diff
@@ -1267,7 +1267,7 @@ int main( int argc, char *argv[] )
 
     len = polarssl_snprintf( (char *) buf, sizeof(buf) - 1, GET_REQUEST,
                     opt.request_page );
-    tail_len = strlen( GET_REQUEST_END );
+    tail_len = (int) strlen( GET_REQUEST_END );
```

Comment:
I looked at the code again and I think that if we change this to size_t we will eventually need to use casts back to int. The problem is that functions such as `snprintf()` return int. Also, note that all the input parameters, etc are of type int so its not that straight forward to change one type. I think its best if we just investigate these problems as part of https://github.com/ARMmbed/mbedtls/issues/1007